megamek/src/megamek/common/planetaryconditions/PlanetaryConditions.java

```diff
@@ -988,4 +990,23 @@ public String getTemperatureIndicator() {
 
         return MSG_INDICATOR_TEMPERATURE_NORMAL;
     }
+
+    @Override
+    public String toString() {
+        return "[PlanetaryConditions]"
+                + ((temperature != 25) ? "; Temperature: " + temperature : "")
+                + ((gravity != 1) ? "; Gravity: " + gravity : "")
+                + (!atmosphere.isStandard() ? "; Pressure:" + atmosphere : "")
```

Comment:
Is this used in serialize it to a file?  

if so, it will break when the language is changed, since atmosphere.toString() returns the value from the resource file for the selected language.  Same with the others.

Reply:
I will never intentionally use toString() for serialization. I add toString() implementations so I see useful info in the IDE when debugging.